python/lsst/ts/mtdome/mock_llc/amcs.py

```diff
@@ -48,6 +53,61 @@
 
 PARK_POSITION = 0.0
 
+SET_ZERO_AZ_ALLOWED_STATES = [
+    MotionState.PARKED.name,
+    InternalMotionState.STATIONARY.name,
+    MotionState.STOPPED.name,
+]
+
+
+def get_distance(start_position: float, end_position: float) -> float:
+    """Determines the smallest distance [rad] between the initial and
+    target positions assuming motion around a circle.
+
+    Parameters
+    ----------
+    start_position : `float`
+        The start position.
+    end_position : `float`
+        The end position.
+
+    Returns
+    -------
+    distance: `float`
+        The smallest distance between the initial and target positions.
```

Comment:
For the physical amounts, please help to add the units. This comment applies to all the code change in this PR.